packages/wrangler/src/deployment-bundle/bundle.ts

```diff
@@ -32,7 +35,7 @@ export type BundleResult = {
 	sourceMapMetadata?: SourceMapMetadata | undefined;
 };
 
-type StaticAssetsConfig =
+export type StaticAssetsConfig =
```

Comment:
I exported it because it is part of the public API of the exported `bundleWorker()` function, and maybe people would want to create an object of that type to pass to that function.